lambda-esm-kinesis-filters-sam/test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+# This script will send all files from the ./events/ folder as messages to stream-lambda-esm-filter Kinesis Data Stream
+# Demo template is expected to be deployed to the AWS CLI's default AWS region
+
+command -v aws >/dev/null 2>&1 || { echo >&2 "This script requires AWS CLI (https://aws.amazon.com/cli/) to be installed.  Aborting."; exit 1; }
+
+stream='stream-lambda-esm-filter'
+
+for f in ./events/*
+do
+    fileName=$(basename "$f")
+    key="${fileName%.*}"
+
+    echo "Sending $key payload..."
+    aws kinesis put-record \
+        --stream-name $stream \
+        --data $(base64 -i "$f") \
+        --partition-key $key
```

Comment:
I'm  getting an error with this part. Looks like Base64 is working but I'm getting :
`console-admin:~/environment/serverless-patterns/lambda-esm-kinesis-filters-sam ((698cd01c...)) $ ./test.sh Sending base-json payload...
usage: aws [options] <command> <subcommand> [<subcommand> ...] [parameters]
To see help text, you can run:

  aws help
  aws <command> help
  aws <command> <subcommand> help

Unknown options: OiAiMjAyMy0wMi0wNlQxMDowMDowMC4wMDAwMDBaIiwKICAgICJ1c2VyIjogewogICAgICAgICJ1, c2VybmFtZSI6ICJla3M6cG9kLWlkZW50aXR5LW11dGF0aW5nLXdlYmhvb2siLAogICAgICAgICJn, cm91cHMiOiBbICJzeXN0ZW06YXV0aGVudGljYXRlZCIgXQogICAgfSwKICAgICJzb3VyY2VJUHMi, OiBbICIxMC4wLjAuMSIgXSwKICAgICJSQkFDIjogZmFsc2UsCiAgICAicmVzcG9uc2VTdGF0dXMi, OnsKICAgICAgICAibWV0YWRhdGEiOnt9LAogICAgICAgICJjb2RlIjoyMDAKICAgIH0KfQo=, IiwKICAgICJsZXZlbCI6ICJSZXF1ZXN0IiwKICAgICJyZXF1ZXN0UmVjZWl2ZWRUaW1lc3RhbXAi`